settings.py

```diff
@@ -208,8 +210,22 @@
     if delivered_lang in PROD_LANGUAGES:
         LANGUAGE_URL_MAP[requested_lang.lower()] = delivered_lang
 
-# Override Django's built-in with our native names
+
+def get_locales():
+    locales = {}
+    file = os.path.join(ROOT, 'kuma', 'languages.json')
```

Comment:
On my filesystem this is at `kuma/core/languages.json`. So I would think you need to add the extra `'core'` in there.